client/src/views/events/events.tsx

```diff
@@ -1,43 +1,24 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Logo from "../../components/Logo";
 import { Card } from "antd";
 import "./events.css";
+import axios from "axios";
 
 const { Meta } = Card;
 
-const eventData = [
-  {
-    title: "Event Title #1",
-    image: "image hosting url",
-    body:
-      "This is an example of an event. It will be a short para and you will be able to type some more stuff into the text.",
-  },
-  {
-    title: "Event Title #2",
-    image: "image hosting url",
-    body:
-      "This is an example of an event. It will be a short para and you will be able to type some more stuff into the text.",
-  },
-  {
-    title: "Event Title #3",
-    image: "image hosting url",
-    body:
-      "This is an example of an event. It will be a short para and you will be able to type some more stuff into the text.",
-  },
-  {
-    title: "Event Title #4",
-    image: "image hosting url",
-    body:
-      "This is an example of an event. It will be a short para and you will be able to type some more stuff into the text.",
-  },
-];
-
 const Events: React.FC<Record<string, never>> = () => {
+  const [eventData, setEvents] = useState([]);
+
+  useEffect(() => {
+    axios.get("http://localhost:5000/events").then((res) => {
```

Comment:
eventually we will make these links env variables that change based wheter we are on PROD or DEV